src/hooks/useAmbassadors.tsx

```diff
@@ -59,12 +59,13 @@ type Ambassador = z.infer<typeof apiAmbassadorSchema>;
 const apiSchema = z.object({
   v3: z
     .record(
+      z.string(),
       // Use nullable here as the fallback for when we fail to parse an ambassador
       apiAmbassadorSchema.nullable().catch((ctx) => {
         console.error(
           "Failed to parse ambassador",
-          ctx.input,
-          ctx.error.message,
+          ctx.value,
+          z.prettifyError(ctx.error),
```

Comment:
`ctx.error` is marked as deprecated, pointing to `ctx.issues`, but I don't see how to pass that into `prettifyError`. Have asked in the Zod Discord.